requirements-dev.txt

```diff
@@ -0,0 +1,21 @@
+-r requirements.txt
+
+# Testing
+pytest>=3.0.4
```

Comment:
yep ... I should have clicked anywhere else on the file. In this PR he added minimum and maximum requirements for most of the packages (except this one!). Very wrongly placed github comment on my side :)